fidesctl/.fides/fidesctl.toml

```diff
@@ -7,5 +7,11 @@ database_name = "fidesctl"
 
 test_database_name = "fidesctl_test"
 
+analytics_id = "b1b202153c5c1f681a944689fccce323"
+
 [cli]
 server_url = "http://fidesctl:8080"
+analytics_id = "5b1837ae7ac208e69dcd33fe4cf75405"
```

Comment:
i would actually rather we store these as CI variables or something, these do feel weird to have as plain text and I'm worried people might inadvertently start using these values by default and cause bad data

Reply:
Cool cool, I think that makes sense to avoid that 👍🏽 

Left it in there as part of the `fidesctl init` output but will get that cleaned up for our uses